operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -166,6 +166,21 @@ public static byte[] decodeFromSecret(Secret secret, String key) {
         return Base64.getDecoder().decode(secret.getData().get(key));
     }
 
+    /**
+     * Decode binary item that holds Private key in PEM format from Kubernetes Secret from base64 into byte array
```

Comment:
```suggestion
     * Decode the binary item in a Kubernetes Secret, which holds a private key in PEM format, from base64 to a byte array.
```
Just a nit, won't it be better this way?

Reply:
yes, that does sound better. Thanks!